packages/radix-ui-themes/src/components/container.tsx

```diff
@@ -15,36 +21,46 @@ interface ContainerProps
     MarginProps,
     LayoutProps,
     ContainerOwnProps {}
-const Container = React.forwardRef<ContainerElement, ContainerProps>((props, forwardedRef) => {
-  const {
-    asChild,
-    children: childrenProp,
-    className,
-    ...containerProps
-  } = extractProps(
-    props,
-    containerPropDefs,
-    layoutPropDefs,
-    deprecatedLayoutPropDefs,
-    marginPropDefs
-  );
+const Container = React.forwardRef<ContainerElement, ContainerProps>(
+  ({ width, minWidth, maxWidth, ...props }, forwardedRef) => {
+    const {
+      asChild,
+      children: childrenProp,
+      className,
+      ...containerProps
+    } = extractProps(
+      props,
+      containerPropDefs,
+      subtract(layoutPropDefs, widthPropDefs),
```

Comment:
why do we need to do this?

Reply:
I guess we don't need to with width props having no default right now, but we'd need to if these prop defs were setting a default value. Should we remove and use this only as needed?